src/RtcDS3231.h

```diff
@@ -354,7 +359,7 @@ template<class T_WIRE_METHOD> class RtcDS3231
         // As it takes (8) right-shifts to register the decimal point (DP) to
         // the right of the 0th bit, the overall word scaling equals 256.
         //
-        // For example, at +/- 25.25�C, concatenated registers <r11h:r12h> =
+        // For example, at +/- 25.25�C, concatenated registers <r11h:r12h> =
```

Comment:
looks like your editor is doing extra work here and removing the degree symbol.
Please correct.